sarracen/disc/utils.py

```diff
@@ -60,6 +60,7 @@ def _bin_particles_by_radius(data: 'SarracenDataFrame',
         Locations of the bin edges.
     """
 
+    origin = _get_origin(origin)
```

Comment:
We should refactor this around a bit. Let's not have any of the arguments be optional and also remove `_get_origin()` from `_bin_particles_by_radius()`.

The disc util functions will choose default values for all of these arguments, if the user did not supply them. So there should be no need to have default values here. It is redundant. Removing them here will avoid code duplication.